sorted_rating.sh

```diff
@@ -0,0 +1,30 @@
+echo enter directory which contains movie folder
+read dir
+final_rate.txt
+for f in $dir/movies/*
+ do
+    oname=${f##*/}
+   # echo $oname
+   name=$(echo ${oname//./+})
+   name=$(echo ${name//_/+})
+   name=$(echo ${name// /+})
+
+   curl -s  http://www.imdb.com/find\?ref_\=nv_sr_fn\&q\=$name\&s\=all |grep -E -0  '/tt\w+' |cut -c72- >out.txt
```

Comment:
Also, try not to have more than 79 characters in the same line. Can you try rewriting this?